README.md

```diff
@@ -2,24 +2,19 @@
 
 Python script to parse the output of "virsh dumpxml" command captured with sosreport
 
-# Install
+# Installation
 
-	pip install xmlvirshparser
+## Install using pip
 
-## Manual install
+	sudo pip install xmlvirshparser
 
-	# Clone this repo
-	git clone https://github.com/ervikrant06/virsh_xml_parser.github
-
-	# Install
-	python setup.py install
-
-# Requirements
+## Installing from cloned repo
+alternatively without root
 
-  (Ignore if installing using `pip`)
-
-	pip install xmltodict
-	pip install prettytable
+	git clone https://github.com/ervikrant06/virsh_xml_parser.github
+	cd virsh_xml_parser
+	python -m pip install -r requirements.txt --user
```

Comment:
This can be removed as we already have setup.py taking care of dependencies.
`requirements.txt` will only add additional overhead. `setup.py` is good.